README.md

```diff
@@ -579,6 +601,21 @@ network_connections:
       dhcp4: yes
 ```
 
+Updating a connection profile by matching interface_path setting:
+
+```yaml
+network_connections:
+  - name: eth0
+    type: ethernet
+    autoconnect: yes
+    interface_path:
+      - pci-0000:00:03.0
```

Comment:
Pleaes explain what this path specification means. Does it only work for interfaces with that path?

Reply:
I explained as:
```yaml
network_connections:
  - name: eth0
    type: ethernet
    autoconnect: yes
    # For PCI devices, the path has the form "pci-$domain:$bus:$device.$function"
    # It will only update the interface with the path "pci-0000:00:03.0"
    interface_path:
      - pci-0000:00:03.0
```